src/feature/flow_chart_panel/FlowChartTabView.tsx

```diff
@@ -229,61 +273,23 @@ const FlowChartTab = () => {
           data-rfinstance={JSON.stringify(nodes)}
         >
           <NodeEditMenu
-            selectedNode={selectedNode}
+            selectedNode={selectAllNodes ? null : selectedNode}
             unSelectedNodes={unSelectedNodes}
             manifestParams={manifestParams}
           />
 
-          <ReactFlow
```

Comment:
(Related to my previous comment) You should put the ReactFlow instance back here and use the dummy component I mentioned, instead of creating a wrapper.